HumanControls/HumanControls.cpp

```diff
@@ -125,12 +135,23 @@ void HumanControls::onPinActivated(int pinNr)
 {
     if (pinNr == m_encoderPinSW)
     {
-        m_menuController.menuPress(m_state, (status == HumanControls::STATUS_ENABLED), m_fireController);
+        m_menuController.menuPress(m_state, (status == HumanControls::STATUS_ENABLED));
     }
     else if (pinNr == m_enablePin)
     {
         m_enableController.setIsEnabled(true);
     }
+    else if (pinNr == m_primePin)
+    {
+        if (status == HumanControls::STATUS_ENABLED)
+        {
+            m_fireController.setIsFireToggled(true);
```

Comment:
Should this function be `m_fireController.setIsPrimed(true)`? Ultimately, when the prime button is pressed, it should just be sending a status update to the robot to set its status to `Primed`

Reply:
Done.